frontend/src/views/WheresMyGene/components/GeneSearchBar/components/ShareButton/style.ts

```diff
@@ -29,11 +28,36 @@ export const StyledButtonIcon = styled(ButtonIcon)`
   height: 30px;
 `;
 
-export const StyledNotification = styled(Notification)`
-  ${LegendWrapper} .MuiAlert-root {
-    position: absolute;
-    top: ${HEADER_HEIGHT_PX}px;
-    right: 0px;
-    z-index: 1;
-  }
+export const StyledNotificationWrapper = styled.div`
```

Comment:
Need to wrap Notification in order for animations to work properly, cannot be set on the Notification component itself